test/test_example.py

```diff
@@ -2,6 +2,19 @@
 from ddt import ddt, data, file_data, unpack
 from test.mycode import larger_than_two, has_three_elements, is_a_greeting
 
+try:
+    import yaml
+except Exception:
+    do_not_have_yaml_support = True
+else:
+    do_not_have_yaml_support = False
```

Comment:
As a matter of preference, I'd prefer to word things positively and avoid *double negative* statements like this one.
`has_yaml_support` makes things more readable. Below you can use `not has_yaml_support` for skipping the unit test...